deep-learning/_posts/2017-09-21-Gated-feedback-refinement-network.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: review
+title: "Gated Feedback Refinement Network for Dense Image Labeling"
+tags: deep-learning segmentation
+author: "Clément Zotti"
+cite:
+    authors: "M-A. Islam, M. Rochan, N-D-B. Bruce, Y. Wang"
+    title: "Gated Feedback Refinement Network for Dense Image Labeling"
+    venue: "CVPR 2017"
+    
+pdf: "http://www.cs.umanitoba.ca/~ywang/papers/cvpr17.pdf"
+---
+
+## Model
+
+This paper is a follow up of [Label Refinement Network](https://arxiv.org/pdf/1703.00551v1.pdf).
+The model is based on a encoding-decoding architecture with skip connections, the encoding is done by a VGG-16 network where only the convolutional layers are used.
+The novelty of their work is the use of gating mechanism in the skip connection.
+
+![](/deep-learning/images/gfrn/fig2.png)
+
+As you can see they use multiple crossentropy loss for each resolution of feature maps on the decoding part. This help the network to extract better information at each resolution.
+
+## Gate Refinement unit
+
+The **Gate unit** merge two resolution of feature maps by upsampling the lower resolution and make an element wise product of the features. These features are finally handled by the **Refinement unit** that produce a label segmentation map.
+
+The main idea behind this is that lower resolution label maps integrate higher-frequency details and that lead to better segmentation maps.
```

Comment:
Are you sure this sentence is right? How can a lower resolution can contain higher-frequency details? It should be the opposite, no?

Reply:
you are right